webapp/src/app/api/pull-request/route.ts

```diff
@@ -92,9 +89,9 @@ export async function POST() {
       auth: GITHUB_AUTH,
       baseUrl: 'https://api.github.com',
       log: {
-        debug: () => {},
-        error: err,
-        info: () => {},
+        debug: debug,
+        error: () => {},
```

Comment:
a bit not sure about this if it will will mute github package when there is an error

Reply:
I think error handling is not a matter of logging so it should be handled elsewhere altogether IMO.